README.md

```diff
@@ -114,6 +114,7 @@ service_bind_address = "0.0.0.0:8081"
 secret_key = "secret_key" # Secret key for the tokens generation
 access_token_lifetime = 1 # In minutes
 refresh_token_lifetime = 1 # In days
+google_client_id="" # Google client id for google login (In case google-oauth feature is enabled)
```

Comment:
The comments are good but we might want to make it more explicilty clear that it is optional